src/stores/simulation.ts

```diff
@@ -106,8 +106,8 @@ export const allResourceTypes: Readable<ResourceType[]> = derived(
   [resourceTypes, externalResources],
   ([$resourceTypes, $externalResources]) => {
     return $resourceTypes
-      .map(({ name, schema }) => ({ name, schema }))
-      .concat($externalResources.map(({ name, schema }) => ({ name, schema })));
+      .map(({ name, schema, description = '' }) => ({ description, name, schema }))
```

Comment:
Because `description` is optional in `ResourceType`, you don't need to define it as `''` here.